journals/apps/journals/utils.py

```diff
@@ -64,6 +70,52 @@ def get_image_url(image, rendition='original'):
         return image.file.url
 
 
+def get_discovery_journal(api_client, uuid=None):
+    """
+        Get Journals form discovery
```

Comment:
typo: form->from

Reply:
might want to consider moving API related utils into a separate file, like api-utils.py or something. Just so api related functions are grouped together.